src/gluonts/model/deepar/_network.py

```diff
@@ -877,10 +944,11 @@ def hybrid_forward(
         past_time_feat: Tensor,
         past_target: Tensor,
         past_observed_values: Tensor,
-        past_is_pad: Optional[Tensor],
+        past_is_pad: Tensor,
         future_time_feat: Tensor,
         future_target: Tensor,
         future_observed_values: Tensor,
+        past_past_feat_dynamic_real: Optional[Tensor] = None,
```

Comment:
past two times? You mean `past_only_feat_dynamic_real` or `past_only_time_feat`? Also docstring needs to be updated for this new argument.

Reply:
Yes this is weird as well. My goal was to use the build-in feature naming from `FieldName` for the `hybrid_forward` call: https://github.com/awslabs/gluon-ts/blob/master/src/gluonts/dataset/field_names.py#L30, which would be `past_feat_dynamic_real`. This then gets turned to `past_past_feat_dynamic_real` in the `InstanceSplitter`https://github.com/awslabs/gluon-ts/blob/master/src/gluonts/transform/split.py#L179. 

One way to circumvent is to have a different channel in the `InstanceSplitter` that does not do the field name modification to that channel (past only time series channel)? 

Do you have other suggestions?